test/conformance/testing/include/uur/fixtures.h

```diff
@@ -98,7 +98,6 @@ struct urDeviceTest : urPlatformTest,
     }
 
     void TearDown() override {
-        EXPECT_SUCCESS(urDeviceRelease(device));
```

Comment:
urDeviceTest doesn't actually own the device handle; it's instead owned by the DevicesEnvironment.

I don't think urDeviceRetain/Release is needed here, since I think DevicesEnvironment must outlive all its tests?

Reply:
Updated.